### PR TITLE
WIP: Added parsing GCP spark cluster state if present

### PR DESCRIPTION
**Pull request by iuliana**

### CODE REVIEW

File: src/main/java/io/cloudsoft/terraform/TerraformSshDriver.java [hunk 1]
```diff
@@ -73,15 +74,16 @@ protected String getLogFileLocation() {
...
         OsDetails os = getLocation().getOsDetails();
         // If no details, assume 64-bit Linux
         if (os == null) return "linux_amd64";
         // If not Mac, assume Linux
         String osType = os.isMac() ? "darwin" : "linux";
-        String archType = os.is64bit() ?
-                os.getArch().toLowerCase().contains("arm") ? "arm64" : "amd64":
-                os.getArch().toLowerCase().contains("arm") ? "arm" : "386";
+        String archType = os.is64bit() ? // temporary, remove after Juan's changes in brooklyn-core
+                Pattern.matches(ARM_ARCH_PATTERNS, os.getArch().toLowerCase()) ? "arm64" : "amd64":
```
Comment (jcabrerizo) [inline src/main/java/io/cloudsoft/terraform/TerraformSshDriver.java:85]: this could be now replaced with `os.isArm`.
don't worry for replace it

**Review by jcabrerizo — APPROVED** [review]
LGTM @iuliana

File: src/main/java/io/cloudsoft/terraform/TerraformSshDriver.java (hunk 1 above)
Comment (iuliana) [reply src/main/java/io/cloudsoft/terraform/TerraformSshDriver.java:85]: @jcabrerizo  make the changes on `main` and send the PR, I will merge it in my branch.

Comment (jcabrerizo) [reply src/main/java/io/cloudsoft/terraform/TerraformSshDriver.java:85]: PR: https://github.com/cloudsoft/brooklyn-terraform/pull/55

Comment (iuliana) [reply src/main/java/io/cloudsoft/terraform/TerraformSshDriver.java:85]: @jcabrerizo thx

**Pull request merged**